code/datums/diseases/advance/symptoms/scalerot.dm

```diff
@@ -0,0 +1,74 @@
+/*
+//////////////////////////////////////
+scalerot
+	Increases resistance.
+	Increases stage speed.
+	Little transmittable.
+	high level
+	other races can be carriers
+Bonus
+	Rots away lizards and flies (chitinous scales) and reverts them to a fleshy form
+//////////////////////////////////////
+*/
+/datum/symptom/scalerot
+		name = "Scale rot"
+		stealth = 2
+		resistance = 3
+		stage_speed = 2
+		transmittable = 1
+		level = 6
+		severity = 5
+
+/datum/symptom/scalerot/Activate(var/datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/mob/living/carbon/human/M = A.affected_mob
```

Comment:
Is the default human only ? If not you probably want to restrict this one from monkeys.